Notebooks/historical_vote_proportions.ipynb

```diff
@@ -0,0 +1,1205 @@
+    "* We will end with a simple example of analysis performed on some voting records data.\n",
+    "* The purpose of this notebook is not to try to explain any patterns or lack thereof in historical vote's, but merely to walk through an example of using CDP's tools and data to perform actual analysis on the data available to any interested party.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***\n",
+    "\n",
+    "<center> <h2> Load relevant libraries for the task</h2> </center>\n",
+    "\n",
+    "***"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Libraries for connection and retrieval from the database\n",
+    "import fireo  \n",
+    "from google.cloud.firestore import Client\n",
+    "from google.auth.credentials import AnonymousCredentials\n",
+    "from cdp_backend.database import models as db_models\n",
+    "\n",
+    "\n",
+    "## Libraries for working with the data files\n",
+    "import pandas as pd\n",
+    "\n",
+    "## For plotting\n",
+    "from matplotlib import pyplot as plt\n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***\n",
+    "\n",
+    "<center> <h2>Connect to the data base</h2> </center>\n",
+    "\n",
+    "***"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## To start off we will connect to Seattle's data base.\n",
+    "seattle = \"cdp-seattle-staging-dbengvtn\"\n",
+    "fireo.connection(client=Client(\n",
+    "    project=seattle,\n",
+    "    credentials=AnonymousCredentials()\n",
+    "))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***\n",
+    "\n",
+    "<center> <h3> Fetch all historical votes from data base</h3> </center>\n",
+    "\n",
+    "***"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[<cdp_backend.database.models.Vote at 0x7f904a56db10>,\n",
+       " <cdp_backend.database.models.Vote at 0x7f904a56dbd0>,\n",
+       " <cdp_backend.database.models.Vote at 0x7f904aa98c50>,\n",
+       " <cdp_backend.database.models.Vote at 0x7f904aa98e90>,\n",
+       " <cdp_backend.database.models.Vote at 0x7f904aa98f90>]"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "############ Select all of the records from the Vote table ############\n",
+    "\n",
+    "## If you want to specify a certain amount of records, just use an argument within fetch() function.\n",
+    "seattle_voting_records = list(db_models.Vote.collection.fetch()) \n",
```

Comment:
I think _technically_ there is a default value to `fetch`. It doesn't try to fetch all of them at once, I think it defaults to 5000? or something?

Reply:
Good point, I added in comments

Reply:
Actually now that you have more data backfilled seattle has just north of 10,000 records and if fetch is left empty it still fetches all of them?